nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -952,7 +952,27 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 			// The object has no text, but we do want to render its children.
 			auto [varChildren, accChildRes] = getAccessibleChildren(pacc, 0, childCount);
 			if (S_OK != accChildRes || varChildren.size() == 0) {
-				LOG_ERROR(L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes);
+				std::wstringstream msg;
+				msg << L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes;
+				switch (accChildRes) {
+				case E_NOINTERFACE:

```

Comment:
nit: Should this all be indented one extra level... or are you deliberately putting the cases at the same level as the switch?

Reply:
This is just the way visual studio formats the code (with `clangFormat`, using VisualStudio style). We don't have some other defined standard, so I figure the default is the best approach. We should really should automatically run `clangFormat` to check for style.